packages/mask/src/components/shared/PersonaListDialog.tsx

```diff
@@ -0,0 +1,324 @@
+        e.preventDefault()
+        e.stopPropagation()
+        copyToClipboard(p.persona.identifier.rawPublicKey)
+        showSnackbar(t('applications_persona_copy'), { variant: 'success' })
+    }
+
+    return open ? (
+        <InjectedDialog
+            disableTitleBorder
+            open={open}
+            classes={{
+                paper: classes.root,
+                dialogTitle: classes.header,
+            }}
+            maxWidth="sm"
+            onClose={onClose}
+            title={t('applications_persona_title')}
+            titleBarIconStyle="close">
+            <DialogContent classes={{ root: classes.content }}>
+                <Stack gap={1.5}>
+                    {personas.map((x) => {
+                        return (
+                            <Stack
+                                key={x.persona.identifier.toText()}
+                                direction="row"
+                                alignItems="center"
+                                gap={1}
+                                onClick={() => onSelectPersona(x)}>
+                                <Box flexGrow={0} position="relative">
+                                    {x.avatar && (
+                                        <Avatar
+                                            src={x.avatar}
+                                            sx={{
+                                                width: 30,
+                                                height: 30,
+                                                display: 'inline-block',
+                                                borderRadius: '50%',
+                                            }}
+                                        />
+                                    )}
+                                    {!x.avatar && <Icons.MenuPersonasActive size={30} />}
+                                    {isSamePersona(currentPersonaIdentifier, x.persona) && (
+                                        <Box className={classes.dot} />
+                                    )}
+                                </Box>
+                                <Stack flexGrow={1}>
+                                    <Typography className={classes.nickname}>
+                                        <Stack
+                                            component="span"
+                                            display="inline-flex"
+                                            direction="row"
+                                            alignItems="center"
+                                            gap={0.25}>
+                                            {x.persona.nickname}
+                                            <>
+                                                {!!x.proof.find(
+                                                    (p) =>
+                                                        p.identity.toLowerCase() ===
```

Comment:
Can we use `isSameProfile` here?

Reply:
This profile is on proof, not `ProfileIdentifier`.